src/GitHub.Exports.Reactive/ViewModels/GitHubPane/IIssueListItemViewModel.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace GitHub.ViewModels.GitHubPane
+{
+    /// <summary>
+    /// Represents an item in the issue list.
+    /// </summary>
+    public interface IIssueListItemViewModel : IIssueListItemViewModelBase
```

Comment:
Should probably rename `(I)IssueListItemViewModelBase` to `(I)IssueishListItemViewModel` as the term "issueish" is usually used to refer to something that can either be an issue or a PR.